src/SFA.DAS.AdminService.Web.Tests/Controllers/CertificateTests/CertificateDelete/GivenIRequestToDeleteCertificate.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Controllers;
+using SFA.DAS.AdminService.Web.ViewModels.CertificateDelete;
+
+namespace SFA.DAS.AdminService.Web.Tests.Controllers.CertificateTests.CertificateDelete
+{
+    public class GivenIRequestToDeleteCertificate : CertificateDeleteQueryBase
+    {
+        private IActionResult _result;
+        private CertificateDeleteController _certificateDeleteController;
```

Comment:
A lot of time this would be given the name _sut as in subject under test to make it obvious what was being tested